### PR TITLE
Support more gdal metadata types in postgres

### PR DESCRIPTION
**Pull request by dbrandenstein**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by dbrandenstein** [comment]
[Alle Gdal Metadata Typen in Postgres unterstützen](https://trello.com/c/hnfwtlfn/1305-alle-gdal-metadata-typen-in-postgres-unterst%C3%BCtzen)

File: services/src/pro/datasets/postgres.rs [hunk 1]
```diff
@@ -332,6 +332,8 @@ where
         Ok(match meta_data {
             MetaDataDefinition::GdalMetaDataRegular(m) => Box::new(m),
             MetaDataDefinition::GdalStatic(m) => Box::new(m),
+            MetaDataDefinition::GdalMetaDataList(m) => Box::new(m),
+            MetaDataDefinition::GdalMetadataNetCdfCf(m) => Box::new(m),
```
Comment (ChristianBeilschmidt) [inline services/src/pro/datasets/postgres.rs:336]: Could you add a test that would fail if I'd create a second PR that just removes these lines?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: services/src/pro/datasets/postgres.rs (hunk 1 above)
Comment (dbrandenstein) [reply services/src/pro/datasets/postgres.rs:336]: https://github.com/geo-engine/geoengine/pull/781/commits/01e46e0041668b6d6a1ceffb856b80b1eedb107c

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**